shape.cc

```diff
@@ -0,0 +1,98 @@
+#include <iostream>
+#include <string>
+#include<cmath>
+#include <cstdlib>
+
+#define PI acos(-1)
+
+
+class Shape {
+  public:
+  Shape(std::string name) : name_{name} {}
+  std::string name() { return name_;}
+
+    virtual double area() {
```

Comment:
```suggestion
    virtual double area() = 0;
```
Pure virtual class has no function definition.